### PR TITLE
More updater tests

### PR DESCRIPTION
**Pull request by SeanTAllen**

### CODE REVIEW

**Comment by SeanTAllen** [comment]
This should not be squashed. Each commit is valid unto itself.

File: corral/cmd/_test_cmd_update.pony [hunk 1]
```diff
@@ -11,40 +11,59 @@ actor _TestCmdUpdate is TestList
...
+  """
+  Verify that when using mutually recursive corral.json files that we
+  execute the correct number of operations
+  """
+
+  fun name(): String =>
+    "cmd/update/" + __loc.type_name()
+
+  fun apply(h: TestHelper) ? =>
+
```
Comment (EpicEric) [inline corral/cmd/_test_cmd_update.pony:41]: Extra line here.

```suggestion
```

File: corral/cmd/_test_cmd_update.pony [hunk 2]
```diff
@@ -10,9 +10,9 @@ actor _TestCmdUpdate is TestList
 
   fun tag tests(test: PonyTest) =>
     test(_TestEmptyDeps)
-    test(_TestRegression120)
     test(_TestMutuallyRecursive)
-
+    test(_TestRegression120)
+    test(_TestSelfReferential)
```
Comment (EpicEric) [inline corral/cmd/_test_cmd_update.pony:15]: Missing a line break.
```suggestion
    test(_TestSelfReferential)

```

**Comment by EpicEric** [comment]
Is there any situation where the numerical values passed to `_OpsRecorder` will differ? So far it seems that all three of them will be the same. But I haven't reviewed what the tests themselves do in-depth.

**Comment by SeanTAllen** [comment]
I'm accepting @EpicEric's changes so, ya let's squash this.

**Comment by SeanTAllen** [comment]
@EpicEric at the moment, I don't believe so, but that is more a result of the current implementation of updater than anything else.

**Comment by SeanTAllen** [comment]
@EpicEric updated

**Comment by EpicEric** [comment]
I see that there are some `lock.json` files still in testdata. Should these be removed as well?

**Comment by SeanTAllen** [comment]
@EpicEric they are old. I think they should be removed as part of another PR.

**Pull request merged**